main.go

```diff
@@ -894,6 +900,8 @@ func parseFlags() (config, error) {
 		"The address on which the internal server listens.")
 	flag.StringVar(&cfg.server.healthcheckURL, "web.healthchecks.url", "http://localhost:8080",
 		"The URL against which to run healthchecks.")
+	flag.BoolVar(&cfg.logs.enableNativeRoutes, "logs.native.routes", false,
+		"Enable native /loki/api/v1 routes for opaque proxying to Loki. Requires passing X-Scope-OrgID header.")
```

Comment:
```suggestion
		"Enable '/loki/api/v1'  prefix instead of '/logs/api/v1/{tenant}/loki/api/v1` prefix for routes to allow opaque proxying to Loki. Requires passing X-Scope-OrgID header.")
```